xvncviewer is an alternative on debian.
vncviewer is what the vnc distribution and we use.

--- avahi/avahi-ui/bssh.c
+++ avahi/avahi-ui/bssh.c
@@ -76,8 +76,8 @@
 
             gtk_widget_destroy(d);
             
-            g_print("xvncviewer %s\n", p);
-            execlp("xvncviewer", "xvncviewer", p, NULL); 
+            g_print("vncviewer %s\n", p);
+            execlp("vncviewer", "vncviewer", p, NULL); 
             
         } else {
             char p[16];
